repeated_solver.py

```diff
@@ -125,62 +101,67 @@
                     filename = "/tmp/" + sketch_name + "_new_sketch_with_hole_value.sk"
                 counter_example_assert = ""
                 counter_example_definition = ""
+                #Add multiple counterexample range from 2 bits to 10 bits
                 for bits in range(2, 10):
                     output_with_counter_example = counter_example_generator(
-                        bits, filename, num_fields_in_prog, num_state_vars)
+                        bits, filename, num_fields_in_prog, num_state_groups)
+                    print(bits)
```

Comment:
Nit: better to print what this is. 
```suggestion
                    print("Bit: %d" % bits)
```